docs/badges.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Badges
-permalink: /badges
+permalink: /badges/
```

Comment:
since you have the file in jekyll root, you don't need this line at all I guess. pls check.